test/unit/reports/organisation_attachments_report_test.rb

```diff
@@ -0,0 +1,42 @@
+require "test_helper"
+
+class Reports::OrganisationAttachmentsReportTest < ActiveSupport::TestCase
+  test "returns a report containing unaccessible attachments for an organisation" do
+    publication = create(:publication, :published, :with_file_attachment)
+    organisation = publication.organisations.last
+
+    path = nil
+    Timecop.freeze do
```

Comment:
Could we move all of the test contents inside this block, and remove the slightly hacky `path = nil` initialisation? Same applies to the other tests.

Reply:
Sure thing